src/scripts/gen-react-bindings.js

```diff
@@ -8,6 +9,40 @@ fs.mkdir(REACT_BINDINGS_DIRECTORY, { recursive: true })
 const COMPONENT_PREFIX = 'leo'
 const SVELTE_REACT_WRAPPER_PATH = '../shared/svelte-react.js'
 
+const getEventsTypeDefinition = (componentName, componentEventNames) => {
+  if (componentEventNames.length === 0) return null
```

Comment:
Might be worth logging a warning if we have an untyped `createEventDispatcher`

Reply:
I am not warning because I rewrote this to use the `/types/*.svelte.d.ts` file, so the script doesn't know if `createEventDispatcher` is typed anymore unfortunately